src/controllers/user.controller.ts

```diff
@@ -51,6 +64,57 @@ class UserController {
 
     res.status(NO_CONTENT).send();
   }
+
+  @SafeController
+  static async login(req: Request, res: Response, next: NextFunction) {
+    passport.authenticate('local', { session: false }, (err: Error, user: User, info: any) => {
+      if (err) {
```

Comment:
If you haven't custom logic you can use the option object to allow passport work in default flow. It will use your strategies